lib/lotus/entity.rb

```diff
@@ -235,6 +252,46 @@ def update(attributes={})
       end
     end
 
+    # Getter for hash of changed attributes.
+    # Return empty hash, if there is no changes
```

Comment:
what do you think about following our conventional `@return [::Hash] the changed attributes`?